packages/eslint-plugin/ROADMAP.md

```diff
@@ -167,6 +174,7 @@ It lists all TSLint rules along side rules from the ESLint ecosystem that are th
 | [`no-reference-import`]             | 🛑  | N/A                                                                                 |
```

Comment:
Could you please add a link to `@typescript-eslint/triple-slash-reference` here

As per #1627